Configuration/Geometry/python/GeometryDD4hepSimDB_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+
+#
+# Geometry master configuration
+#
+# Ideal geometry, needed for simulation
+
+DDSpecParRegistryESProducer = cms.ESProducer("DDSpecParRegistryESProducer",
+                                             appendToDataLabel = cms.string('')
+)
+
+DDVectorRegistryESProducer = cms.ESProducer("DDVectorRegistryESProducer",
+                                            appendToDataLabel = cms.string(''))
+
+DDCompactViewESProducer = cms.ESProducer("DDCompactViewESProducer",
+                                         appendToDataLabel = cms.string('')
+)
+
+from DetectorDescription.DDCMS.DDDetectorESProducerFromDB_cfi import *
+
+from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
+
+from Geometry.EcalCommonData.ecalSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalSimDBConstants_cff         import *
+
+from Geometry.MuonNumbering.muonGeometryConstants_cff       import *
+from Geometry.MuonNumbering.muonOffsetESProducer_cff        import *
```

Comment:
Do you understand why this muonOffset is needed for DD4hepSimDB only, but not DDDSimDB? I understand that GeometryDDDSimDB is a clone of previous GeometrySimDB_cff.py, but it seems that the call should be done both.

Reply:
I think `MuonOffsetESProducer` is the DD4hep replacement for the DDD `MuonNumberingInitialization`. It iwould be redundant in the DDD config, which we know works correctly as it is.

Reply:
Having MuonOffsetESProducer does not do any harm for DDD. It is present when the xml files are read in. It may be worthwhile to make the two versions as symmetric as possible.

Reply:
If it ain't broke, don't fix it. The DDD config has been working correctly for years. I would be worried about changing it, even if the change SHOULD be harmless. If the change is necessary, it would be better to do in a separate PR, so if any issues arise, it would be easier to identify which changes caused the issues, rather than mixing them together in this PR. These configs will be heavily used by central production, so I think it is worthwhile to be over-cautious in this case.

Reply:
As far as I can tell, the muon offset map is used only in the DD4hep case. See the code here [1]. It would _probably_ be harmless to include it in the DDD config, but it would serve no purpose and could have an unforeseen side effect.

[1]: https://cmssdt.cern.ch/lxr/source/SimG4CMS/Muon/src/MuonG4Numbering.cc?v=CMSSW_12_0_X_2021-07-01-2300#0055

Reply:
I agree, if needed we should do in the separate PR. I just try to learn the difference. Thanks @cvuosalo @bsunanda